src/libpmempool/replica.h

```diff
@@ -62,6 +68,16 @@
  */
 #define REMOTE_NLANES	1
 
+/*
+ * Helping structures for storing part's health status
+ */
+struct part_health_status {
+	unsigned flags;
+	struct badblocks bbs;		/* structure with bad blocks */
+	char *recovery_file;		/* name of bad blocks recovery file */
+	int exist_recovery_file;	/* bad blocks recovery file exists */
```

Comment:
`recovery_file_exists`
but do we need this variable? isn't `recovery_file != NULL` sufficient?